src/esp/assets/ResourceManager.cpp

```diff
@@ -123,16 +124,26 @@ ResourceManager::~ResourceManager() {
 }
 
 void ResourceManager::buildImporters() {
+  // Preferred plugins, Basis target GPU format
+  importerManager_.setPreferredPlugins("GltfImporter", {"CgltfImporter"});
+#ifdef ESP_BUILD_ASSIMP_SUPPORT
+  importerManager_.setPreferredPlugins("ObjImporter", {"AssimpImporter"});
+  Cr::PluginManager::PluginMetadata* const assimpmetadata =
+      importerManager_.metadata("AssimpImporter");
+  assimpmetadata->configuration().setValue("ImportColladaIgnoreUpDirection",
```

Comment:
Hmm, I wish there was a way to use this if it wasn't specified in the JSON...

Reply:
Could you explain what you mean? Thanks :)

Reply:
I am worried the fact that other importers like Blender may support Collada's up direction may cause confusion for end-users when the rotations do not match. Seems like it's harmful to ignore an up direction defined in the dae. What about other formats that have their own up direction, it feels like we should try to be consistent with the handling of the up direction across formats. (For instance, the rotation in the dataset should be relative to the up direction in the mesh file).